airflow/utils/state.py

```diff
@@ -96,17 +127,17 @@ def color_fg(cls, state):
             return 'white'
         return 'black'
 
-    running = frozenset([RUNNING, SENSING])
+    running: FrozenSet[TaskInstanceState] = frozenset([TaskInstanceState.RUNNING, TaskInstanceState.SENSING])
```

Comment:
Huh, surprised this type hint is needed.

Reply:
It may not be, I just like adding explicit type hints where I think it helps the reader of the code understand what's going on.

Reply:
Yep. I also often do that even if not technically needed to make it explicit. I think code should be much more optimised for reader's time than writer's time to write it ;).